### ISSUE TITLE
namespace not being logged when using fetch_multi

### ISSUE THREAD

**Issue by fbjork** [petergoldstein/dalli#551]
namespace isn't logged when using fetch_multi.

```ruby
Rails.cache.fetch_multi(1, 2, namespace: "foo")
```

logs the following:

```
Cache fetch_multi: [1, 2]
```

I would expect namespace to be logged as well.

**Comment by petergoldstein** [comment]
I believe this should be addressed by #511

**Comment by petergoldstein** [comment]
Closing as #511 is merged.  This fix will be included in 2.7.5
